app/models/mixins/aggregation_mixin/methods.rb

```diff
@@ -39,11 +39,11 @@ def all_storages
     end
 
     def aggregate_hardware(from, field, targets = nil)
-      from      = from.to_s.singularize
+      from      = from.to_s.singularize.to_sym
```

Comment:
not sure if the `to_sym` is needed. maybe the to sym is helping us with the rubocop?